Documentation/Installation/Deployer/Index.rst

```diff
@@ -1,17 +1,90 @@
-.. include:: /Includes.rst.txt
+..  include:: /Includes.rst.txt
 
-.. index:: Deployment; Deployer
+..  index:: Deployment; Deployer
 
-.. _deployment-deployer:
+..  _deployment-deployer:
 
 ========
 Deployer
 ========
 
-`Deployer <https://deployer.org>`__ is a deployment tool
-written in PHP. A full description about how to get deployer running
-for TYPO3 can be found at https://t3terminal.com/blog/typo3-deploy/
+`Deployer <https://deployer.org/>`__ is a deployment tool written in PHP. Please
+also have a look into the
+`documentation of Deployer <https://deployer.org/docs/7.x/installation>`__,
+mainly the "Basics" and "Main Concepts".
 
-..  literalinclude:: _deploy.php
+..  todo: Explain:
+    * releases (which are symlinked)
+
+.. contents:: **Table of Contents**
+   :local:
+
+
+Installation
```

Comment:
Maybe some preface here like:

Configurations created with Deployer are called "recipes", and they contain the steps to perform for your deployment.

A usual deployment performs these steps:

* Initialize configuration
* Check out the project's repository in a (temporary, project-independent) workspace
* Install the project dependencies (including TYPO3) with Composer
* Perform possible extra steps like running JavaScript frontend builds, fetching assets
* Collect all created files and transfer them to the target server, ensuring proper directory structure
* Execute remote commands on the target server (for example flushing the cache via the :file:`typo3` CLI binary

(Just a stub, not sure what I might be missing)